Tools/WinMLRunner/src/BindingUtilities.h

```diff
@@ -494,4 +515,89 @@ namespace BindingUtilities
             std::cout << std::endl;
         }
     }
+
+    void SaveEvaluationResults(const LearningModel &model, const CommandLineArgs &args, const IMapView<hstring, winrt::Windows::Foundation::IInspectable>& results, OutputHelper &output, uint32_t iterationNum)
+    {
+        std::string iterationResult;
+        size_t hash = 0;
+
+        for (auto&& desc : model.OutputFeatures())
+        {
+            if (desc.Kind() == LearningModelFeatureKind::Tensor)
+            {
+                std::wstring name(desc.Name());
+                std::string tempName;
+                for (int i = 0; i < name.size(); i++)
+                {
+                    if (isalnum(name[i]))
+                    {
+                        tempName += name[i];
+                    }
+                }
+                TensorFeatureDescriptor tensorDescriptor = desc.as<TensorFeatureDescriptor>();
+                TensorKind tensorKind = tensorDescriptor.TensorKind();
+
+                switch (tensorKind)
+                {
+                case TensorKind::String:
+                {
+					std::cout << "**********WARNING**********\nTensorKind is of type String\nOutput Tensor not saved";
+                }
+                break;
+                
+                case TensorKind::Float:
+                {
+                    com_ptr<ITensorNative> itn = results.Lookup(desc.Name()).as<ITensorNative>();
+                    float* Tensor;
+                    uint32_t uCapacity;
+                    HRESULT(itn->GetBuffer(reinterpret_cast<BYTE**>(&Tensor), &uCapacity));
+                    hash = hash_data(Tensor, uCapacity);
+
+                    UINT maxIndex = 0;
+                    auto maxValue = *Tensor;
+                    UINT size = uCapacity / sizeof(float_t);
+                    for (UINT i = 0; i < size; i++)
+                    {
+                        float val = *(Tensor+i);
+                        if (maxValue < val)
+                        {
+                            maxValue = val;
+                            maxIndex = i;
+                        }
+                    }
+                    iterationResult = "Index: " + std::to_string(maxIndex) + "; Value: " + std::to_string(maxValue);
+                    output.WriteTensorResultToCSV(Tensor, iterationNum, args, uCapacity, tempName);
+                }
+                break;
+
+                case TensorKind::Int64:
```

Comment:
Can we add a test to test this case?

Reply:
Is there a model that we can test for Int64? To verify that this code path works?